ra-optimisation/linear-rao/src/main/java/com/farao_community/farao/linear_rao/LinearRaoData.java

```diff
@@ -4,49 +4,267 @@
  *  License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/.
  */
-
 package com.farao_community.farao.linear_rao;
 
 import com.farao_community.farao.commons.FaraoException;
-import com.farao_community.farao.data.crac_api.*;
+import com.farao_community.farao.data.crac_api.Cnec;
+import com.farao_community.farao.data.crac_api.Crac;
+import com.farao_community.farao.data.crac_api.PstRange;
+import com.farao_community.farao.data.crac_api.RangeAction;
+import com.farao_community.farao.data.crac_result_extensions.*;
 import com.farao_community.farao.util.SystematicSensitivityAnalysisResult;
 import com.powsybl.iidm.network.Network;
 
+import java.util.*;
+
 /**
- * @author Pengbo Wang {@literal <pengbo.wang at rte-international.com>}
+ * A LinearRaoData is an object that gathers Network, Crac and SystematicSensitivityAnalysisResult data. It manages
+ * variants of these objects to ensure data consistency at any moment. Network will remain the same at any moment
+ * with no variant management to be able to use this object in a multi-thread environment. It is a single point of
```

Comment:
We don't create variants "to be able to use this object in a multti-thread envinronment" is conter-intuitive here. I would just delete this sentence.